tests/capability_set_ops.c

```diff
@@ -176,7 +177,7 @@ int main(int argc, char *argv[])
             uint32_t n;
 
             for (n = 0; n < num_capability_sets[i]; n++) {
-                printf("Device %d - Set %d:\n", i, n);
+                printf("Device %" PRId64 " - Set %d:\n", i, n);
```

Comment:
Technically this is still wrong.  It should have been %zd, but it might just be easier (and more portable in practice) to make "i" and "num_devices" an int.